FSMS.Services/FileActions/SummarizeFileAction.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FSMS.Core.Attributes;
+using FSMS.Core.Interfaces;
+
+namespace FSMS.Services.FileActions;
+
+[FileAction("summary", ".txt")]
+public class SummarizeFileAction : IFileAction
+{
+    public bool CanHandle(string actionName, string filePath)
+    {
+        var extension = Path.GetExtension(filePath);
+        return actionName.Equals("summary", StringComparison.OrdinalIgnoreCase) &&
+               extension.Equals(".txt", StringComparison.OrdinalIgnoreCase);
+    }
```

Comment:
Why do you need to duplicate command name and extensions in every action in CanHandle method? I think it must be single place where you provide such a logic